README.md

```diff
@@ -16,8 +16,23 @@ To see a current list of commands, run `make help`.
 
 `UNPAYWALL_EMAIL`: email address to include in API call as required in their [documentation](https://unpaywall.org/products/api). Your personal email is appropriate for development. Deployed and for tests, use the timdex moira list email.
 
+#### Authentication
+
+Access to some of the config values below is limited. Please contact someone in the EngX team if you need help locating
+them.
+
+`BASE_URL`: The base url for the app. This is required for Omniauth config.
+`FAKE_AUTH_CONFIG`: Switches Omniauth to developer mode when set to `true`. Fake auth is also enabled whenever the
```

Comment:
I believe this should be `FAKE_AUTH_ENABLED` and it also reads like it is not required in PR builds, but in my testing I had to set `FAKE_AUTH_ENABLED` to true in the PR build to switch the Sign In button to fake auth